root/edit/CannotApproveEdit.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import EditLink from '../static/scripts/common/components/EditLink';
+import {
+  EDIT_STATUS_DELETED,
+  EDIT_STATUS_TOBEDELETED,
+} from '../constants';
+
+type Props = {
+  +$c: CatalystContextT,
+  +edit: EditT,
+};
+
+const CannotApproveEdit = (
+  {$c, edit}: Props,
+): React.Element<typeof Layout> => {
+  const editDisplay = 'edit #' + edit.id;
+  const editLink = <EditLink content={editDisplay} edit={edit} />;
+  const editIsClosed = !edit.is_open;
+  const editIsCancelled = edit.status === EDIT_STATUS_DELETED;
+  const editBeingCancelled = edit.status === EDIT_STATUS_TOBEDELETED;
+  const reason = editIsCancelled ? (
+    l('The edit has been cancelled.')
+  ) : editBeingCancelled ? (
+    l('The edit is already in the process of being cancelled.')
```

Comment:
Thanks, maybe [the message for the same case in `root/utility/edit.js`](https://github.com/metabrainz/musicbrainz-server/blob/v-2020-08-10/root/utility/edit.js#L86) should be changed at the same time?

Reply:
I guess we might. Made that change in a separate commit.

Reply:
The `TOBEDELETED` status is unused. That was from before MBS-777 when edits weren't cancelled immediately. So I think these messages should be removed instead of modified.